src/migrator/sync.ts

```diff
@@ -69,10 +68,10 @@ const $run = async (
 
 	// migrations only run if the model has been executed before,
 	// to make changes that can't be automatically applied
-	const exists = await checkModelAlreadyExists(tx, modelName);
+	const exists = await sbvrUtils.isModelNew(tx, modelName);
 	if (!exists) {
 		(sbvrUtils.api.migrations?.logger.info ?? console.info)(
-			'First time model has executed, skipping migrations',
+			`First time model ${modelName} has executed, skipping migrations`,
```

Comment:
I find putting the variable part in quotes can help make it clear that you're quoting some user input
```suggestion
			`First time model '${modelName}' has executed, skipping migrations`,
```